nixos/modules/services/torrent/transmission.nix

```diff
@@ -100,56 +115,180 @@ in
         default = "transmission";
         description = "Group account under which Transmission runs.";
       };
+
+      credentialsFile = mkOption {
+        type = types.path;
+        description = ''
```

Comment:
While we generally aren't very good (or consistent) at it, it might be worth mentioning that the file must be readable by the `transmission` user. That isn't always a given.

Reply:
@aanderse, nice catch, I've changed to use `ExecStartPre=+` to run as root.